package.json

```diff
@@ -47,6 +47,10 @@
     "create-react-class": "15.5.3",
     "creditcards": "2.1.2",
     "cross-env": "4.0.0",
+    "d3-array": "1.2.0",
```

Comment:
@greenafrican the other package changes are expected, no worries. redoing the shrinkwrap repins everything that's not pinned to an exact version, so there's always a little extra churn.